nupic/utils.py

```diff
@@ -85,16 +85,15 @@ def getSlidingWindow(self):
     return self.slidingWindow
 
 
-  def __setstate__(self, state):
```

Comment:
I don't think this is true. Until capnp is implemented for entire CLAModel we will be using old version. Then there will be deprecation period. So can't break old version for a while still.